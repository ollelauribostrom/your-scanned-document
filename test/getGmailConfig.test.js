const path = require('path');
const getGmailConfig = require('../src/utils/getGmailConfig');

jest.mock('path');

describe('Tests for getGmailConfig', () => {
  it('returns a config object', () => {
    path.resolve = jest.fn(arg => arg);
    expect(getGmailConfig()).toMatchSnapshot();
  });
});
