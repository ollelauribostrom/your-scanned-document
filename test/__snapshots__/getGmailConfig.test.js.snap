// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for getGmailConfig returns a config object 1`] = `
Object {
  "credentialsPath": "./env/gmail-credentials.json",
  "historyPath": "./env/history.json",
  "pubsubOptions": Object {
    "keyFilename": "./env/service-credentials.json",
    "projectId": "your-scanned-doc-1547418741445",
    "subscriptionName": "projects/your-scanned-doc-1547418741445/subscriptions/email-sub",
    "topicName": "projects/your-scanned-doc-1547418741445/topics/gmail",
  },
  "scope": Array [
    "https://www.googleapis.com/auth/gmail.readonly",
  ],
  "tokenPath": "./env/token.json",
}
`;
